tiered-storage/file-system/src/test/java/org/apache/bookkeeper/mledger/offload/filesystem/impl/FileSystemManagedLedgerOffloaderTest.java

```diff
@@ -111,6 +112,31 @@ public void testOffloadAndRead() throws Exception {
         }
     }
 
+    @Test
+    public void testOffloadAndReadMetrics() throws Exception {
+        LedgerOffloader offloader = fileSystemManagedLedgerOffloader;
+        UUID uuid = UUID.randomUUID();
+        offloader.offload(toWrite, uuid, map).get();
+
+        LedgerOffloaderMXBeanImpl mbean = (LedgerOffloaderMXBeanImpl)offloader.getStats();
+        assertTrue(mbean.getOffloadErrors(topic) == 0);
+        assertTrue(mbean.getOffloadBytes(topic) > 0 );
+        assertTrue(mbean.getReadLedgerLatencyBuckets(topic).getCount() > 0);
+        assertTrue(mbean.getWriteToStorageErrors(topic) == 0);
+
+        ReadHandle toTest = offloader.readOffloaded(toWrite.getId(), uuid, map).get();
+        LedgerEntries toTestEntries = toTest.read(0, numberOfEntries - 1);
+        Iterator<LedgerEntry> toTestIter = toTestEntries.iterator();
+        while(toTestIter.hasNext()) {
+            LedgerEntry toTestEntry = toTestIter.next();
```

Comment:
Can we compare the read entries size is equals the `mbean.getReadOffloadBytes(topic)`?

Reply:
already updated